modules/friendlytalkback.js

```diff
@@ -95,28 +95,17 @@ Twinkle.talkback.callback = function() {
 	wpapi.post();
 };
 
-Twinkle.talkback.optout = null;
+Twinkle.talkback.optout = '';
 
 Twinkle.talkback.callback.optoutStatus = function(apiobj) {
-	var xml = apiobj.getXML();
-	var $el = $(xml).find('el');
-
+	var $el = $(apiobj.getXML()).find('el');
 	if ($el.length) {
 		Twinkle.talkback.optout = mw.config.get('wgRelevantUserName') + ' prefers not to receive talkbacks';
 		var url = $el.text();
-		if (url.indexOf('reason=') > -1) {
-			Twinkle.talkback.optout += ': ' + decodeURIComponent(url.substring(url.indexOf('reason=') + 7));
-		} else {
-			Twinkle.talkback.optout += '.';
-		}
-	} else {
-		Twinkle.talkback.optout = false;
-	}
-
-	var $status = $('#twinkle-talkback-optout-message');
-	if ($status.length && Twinkle.talkback.optout) {
```

Comment:
both checks are unnecessary now: (i) trying to do `.text()` on empty jquery selection doesn't give any error. (ii) If not opted out, `Twinkle.talkback.optout` is just the empty string - no harm in appending it.